lstchain/io/lstcontainers.py

```diff
@@ -89,6 +93,14 @@ class DL1ParametersContainer(Container):
     tel_pos_y = Field(None, "Telescope y position in the ground")
     tel_pos_z = Field(None, "Telescope z position in the ground")
 
+    trigger_type = Field(None, "trigger type")
+    ucts_trigger_type = Field(None, "UCTS trigger type")
+    trigger_time = Field(None, "trigger time")
```

Comment:
How is this trigger time different from the `gps_time`, `dragon_time` or `ucts_time` from above?

Reply:
From above, where precisely?

Reply:
The same container has these fields, a couple of lines above the ones you added here.

Reply:
If you view this comments in the "changes" tab of github, you can see them

Reply:
yes, now a remember :-) I didn't  add them, I simply  took them out from the ExtraImageInfo because they better belong to the event parameters. Don't ask me to explain why we have so many times and trigger types, that would be a pain for both ;-)

Reply:
But how exactly is the generic `trigger_time` related to the 3 specific (gps, ucts, dragon) and why is it needed as additional field here? This should be in the description, otherwise no one can actually understand those descriptions...